sourcemap/store_test.go

```diff
@@ -142,6 +153,145 @@ func TestStore_Fetch(t *testing.T) {
 	})
 }
 
+func TestFetchTimeout(t *testing.T) {
+	var (
+		errs int64
+
+		apikey  = "supersecret"
+		name    = "webapp"
+		version = "1.0.0"
+		path    = "/my/path/to/bundle.js.map"
+		c       = http.DefaultClient
+		waitc   = make(chan struct{})
+	)
+	ts := httptest.NewServer(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		<-waitc
+		// zlib compress
+		wr := zlib.NewWriter(w)
+		defer wr.Close()
+		wr.Write([]byte(test.ValidSourcemap))
+	}))
+	defer ts.Close()
```

Comment:
unless we need to test a non-timeout after a timeout, this handler could just be `<-req.Context().Done()`

Reply:
which very helpfully pointed out I wasn't attaching the passed-in context to the request :)